### DISCUSSION TITLE
Selected Row Count

### DISCUSSION THREAD

**Discussion by Maglorx** [dbeaver/dbeaver#18523] [Q&A] [answered]
### Description

Hi,

After the last dbeaver update (Version 22.3.0.202212131935), I can’t see the selected row count in the result panel. Is there any option for bringing it back?
![dbaver](https://user-images.githubusercontent.com/45281149/208032122-88f3f6c3-6ed1-43d2-88b2-46626ba5cf6d.jpg)

### DBeaver Version

Version 22.3.0.202212131935

### Operating System

windows 10

**Comment by ShadelessFox** [comment, marked as answer]
Hello @Maglorx,

These options were hidden by default.

You can re-enable them here:

![image](https://user-images.githubusercontent.com/35821147/208100503-51376abc-32b8-431c-8b8b-3ccbd1ea68bc.png)

**Reply by nacesprin** [reply]
Where is it in v25?

![image](https://github.com/user-attachments/assets/52aba421-eec6-429e-bd06-957654b12497)

![image](https://github.com/user-attachments/assets/b1df140b-a905-42de-82c3-d51df30f51e1)

**Reply by nacesprin** [reply]
SOLVED!
It were in Layout option:
![image](https://github.com/user-attachments/assets/7a329eaf-9353-4737-9956-bc47bbf9483d)
